### DISCUSSION TITLE
https://galaxyharvester.net

### DISCUSSION THREAD

**Discussion by wowakan** [pwillworth/galaxyharvester#52] [General]
Hello Sir,

I did add our new server to you website. "New Beginnings". Yes we have our website up but still working on it. Things we are working on.

1. Getting the website, MyBB forums username and password to link up with the Game server. (Done)
2. We have the launcher 90% done. Link to the forums where the login is for website to game: https://swgnewbeginnings.com/forums/
3. While yes we are waiting till you add us to the website, wee could REALLY use some help on how we automate the whole server to upload the Data to your website please.
4. While yes the live server is up we are also doing lots of beta testing on the test server.

Kind Regards,
R. Baker

**Comment by pwillworth** [comment]
Hi, I just activated your galaxy submission.

If you want to automate resource updates, people usually write a script that interacts with the galaxy harvester web service endpoints that runs periodically.  You can find information about how to call the services here in the wiki section [https://github.com/pwillworth/galaxyharvester/wiki/Web-Services](https://github.com/pwillworth/galaxyharvester/wiki/Web-Services)

**Comment by wowakan** [comment]
Thank you sir
